### ISSUE TITLE
Table nested - problems when publishing/unpublihsing Items

### ISSUE THREAD

**Issue by PhocaCz** [joomla/joomla-cms#15662]
### Steps to reproduce the issue

Install extension, e.g. Phoca Guestbook, add a post message and publish/unpublish it in administration. You get success message but nothing changes.

It worked since version Joomla! 3.0.0 - to version 3.6.4. In version 3.6.5 there was this change made:

file:
libraries\joomla\table\nested.php
line: cca 1022 
FROM:
```
// Update and cascade the publishing state.
			$query2->clear()
				->update($this->_db->quoteName($this->_tbl))
				->set('published = ' . (int) $state)
				->where('(lft > ' . (int) $node->lft . ' AND rgt < ' . (int) $node->rgt . ') OR ' . $k . ' = ' . (int) $pk);
			$this->_db->setQuery($query)->execute();
```
TO:
`$this->recursiveUpdatePublishedColumn($pk, $state);`

### Expected result
Item will be published/unpublished

### Actual result
Success is returned but no action done

### System information (as much as possible)
Joomla! 3.6.4 - OK
Joomla! 3.6.5 - 3.0.7 - NOT OK

PHP7

**Comment by csthomas** [comment]
The quick solution is to run a query:
```sql
UPDATE `#__phocaguestbook_items` SET `published` = '1' WHERE `j37_phocaguestbook_items`.`id` = 1;
```

The new code requires to have published column  equal to `1` for ROOT row.

**Comment by PhocaCz** [comment]
Hi, thank you for the info, yes I see it now. The root must be published since Joomla! 3.6.5 - I am working now on update.

When the root is published, then standard method to publish/unpublish the items through nested table class (including testing of parent elements, etc.) works again.

So this can be closed (for other developers: since Joomla! 3.6.5 root level must be set as "publish"

Thank you.
